app/assets/stylesheets/local/forms.scss

```diff
@@ -83,35 +83,37 @@ label[for="steps_abduction_risk_details_form_risk_details"]
 
   & label .heading-small {
     font-weight: normal;
+    margin-top: 0;
   }
 }
 
 .form-block.or-block {
   float: none;
   margin-top: -4rem;
+
   .or-block__shift & {
     margin-top: -4rem;
   }
 }
 
 .form-label {
   font-weight: bold;
-}
 
-.form-label-normal {
-  font-weight: normal;
+  &-normal {
```

Comment:
I would probably avoid joining this with an `&` and keep them separate. It can make it more confusing to find that particular selector using a search.

Reply:
Changes also addressed in a fixup commit :)